src/workerd/api/wpt/url-test.js

```diff
@@ -2,20 +2,138 @@
+    'response.formData() with input: †&†=x',
+    'request.formData() with input: %C2',
+    'response.formData() with input: %C2',
+    'request.formData() with input: %C2x',
+    'response.formData() with input: %C2x',
+    'request.formData() with input: _charset_=windows-1252&test=%C2x',
+    'response.formData() with input: _charset_=windows-1252&test=%C2x',
+    'request.formData() with input: ',
+    'response.formData() with input: ',
+    'request.formData() with input: a',
+    'response.formData() with input: a',
+    'request.formData() with input: a=b',
+    'response.formData() with input: a=b',
+    'request.formData() with input: a=',
+    'response.formData() with input: a=',
+    'request.formData() with input: =b',
+    'response.formData() with input: =b',
+    'request.formData() with input: &',
+    'response.formData() with input: &',
+    'request.formData() with input: &a',
+    'response.formData() with input: &a',
+    'request.formData() with input: a&',
+    'response.formData() with input: a&',
+    'request.formData() with input: a&a',
+    'response.formData() with input: a&a',
+    'request.formData() with input: a&b&c',
+    'response.formData() with input: a&b&c',
+    'request.formData() with input: a=b&c=d',
+    'response.formData() with input: a=b&c=d',
+    'request.formData() with input: a=b&c=d&',
+    'response.formData() with input: a=b&c=d&',
+    'request.formData() with input: &&&a=b&&&&c=d&',
+    'response.formData() with input: &&&a=b&&&&c=d&',
+    'request.formData() with input: a=a&a=b&a=c',
+    'response.formData() with input: a=a&a=b&a=c',
+    'request.formData() with input: a==a',
+    'response.formData() with input: a==a',
+    'request.formData() with input: a=a+b+c+d',
+    'response.formData() with input: a=a+b+c+d',
+    'request.formData() with input: %=a',
+    'response.formData() with input: %=a',
+    'request.formData() with input: %a=a',
+    'response.formData() with input: %a=a',
+    'request.formData() with input: %a_=a',
+    'response.formData() with input: %a_=a',
+    'request.formData() with input: %61=a',
+    'response.formData() with input: %61=a',
+    'request.formData() with input: %61+%4d%4D=',
+    'response.formData() with input: %61+%4d%4D=',
+    'request.formData() with input: id=0&value=%',
+    'response.formData() with input: id=0&value=%',
+    'request.formData() with input: b=%2sf%2a',
+    'response.formData() with input: b=%2sf%2a',
+    'request.formData() with input: b=%2%2af%2a',
+    'response.formData() with input: b=%2%2af%2a',
+    'request.formData() with input: b=%%2a',
+    'response.formData() with input: b=%%2a',
+  ],
+});
+
+export const urlSearchParamsAppend = run('urlsearchparams-append.any.js');
+
+export const urlSearchParamsConstructor = run(
+  'urlsearchparams-constructor.any.js',
+  {
+    expectedFailures: [
+      // TODO(soon): It's supposed to throw when we call new URLSearchParams(DOMException)
```

Comment:
The test names are explanatory enough. We don't need such comments unless we deliberately disable the rule due to a limitation of workerd.